tests/parsers/ps_transcript.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""Tests for PowerShell transcript parser"""
+
+import unittest
+
+from plaso.lib import errors
+from plaso.parsers import ps_transcript
+
+from tests.parsers import test_lib
+
+
+class PSTranscriptParserTest(test_lib.ParserTestCase):
+  """Tests for the PowerShell Transcript parser"""
+
+  def testParse(self):
+    """Tests the parse function on an example file."""
+    parser = ps_transcript.PSTranscriptParser()
+    storage_writer = self._ParseFile(
+        ['powershell_transcript.txt'], parser)
+
+    number_of_events = storage_writer.GetNumberOfAttributeContainers('event')
+    self.assertEqual(number_of_events, 2)
+
+    number_of_warnings = storage_writer.GetNumberOfAttributeContainers(
+        'extraction_warning')
+    self.assertEqual(number_of_warnings, 3)
+
+    number_of_warnings = storage_writer.GetNumberOfAttributeContainers(
+        'recovery_warning')
+    self.assertEqual(number_of_warnings, 0)
+
+    events = list(storage_writer.GetSortedEvents())
+
+    # Test a regular event.
+    expected_event_values = {
+        'build_version': '10.0.17763.1852',
+        'clr_version': '4.0.30319.42000',
+        'command': 'PS C:\\Windows\\system32> whoami; msedgewin10\\ieuser',
+        'compatible_versions': '1.0, 2.0, 3.0, 4.0, 5.0, 5.1.17763.1852',
+        'date_time': '2022-07-21 02:37:49',
+        'data_type': 'ps:transcript:event',
+        'edition': 'Desktop',
+        'host_application': 'C:\\Windows\\System32\\WindowsPowerShell\\v1.0'\
```

Comment:
Per style guide, if possible please avoid using \ to split strings.
Maybe try this, or similar:
```
'host_application': 
    'C:\\Windows\\System32\\WindowsPowerShell\\v1.0'\\powershell.exe',
```